linux_os/guide/services/base/service_cpuspeed_disabled/rule.yml

```diff
@@ -15,13 +15,15 @@ rationale: |-
     but may also apply to server or desktop environments where conserving power is
     highly desirable or necessary.
 
-severity: unknown
+severity: low
 
 identifiers:
     cce@rhel6: 26973-8
+    cce@rhel7: 82383-1
+    cce@rhel8: 82384-9
```

Comment:
@shawndwells we should return the rhel7 and rhel8 CCE back to the pool. Starting with RHEL7, there is no more `cpuspeed` package.

Reply:
Done & removed from prodtype.